.github/workflows/edr-benchmark.yml

```diff
@@ -0,0 +1,86 @@
+name: EDR Benchmark
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/workflows/edr-benchmark.yml"
+      - "rust-toolchain"
+      - "Cargo.lock"
+      - "Cargo.toml"
+      - "crates/**"
+  pull_request:
+    branches:
+      - "**"
+    paths:
+      - ".github/workflows/edr-benchmark.yml"
+      - "rust-toolchain"
+      - "Cargo.lock"
+      - "Cargo.toml"
+      - "crates/**"
```

Comment:
Only running for changes to EDR since it's very slow and there is only one runner + we'll do this anyway once EDR is a separate repo